src/fal/planner/executor.py

```diff
@@ -24,22 +24,30 @@ class State(Enum):
     POST_HOOKS = auto()
 
 
-def _collect_models(groups: List[TaskGroup]) -> Iterator[str]:
+def _collect_nodes(groups: List[TaskGroup], fal_dbt: FalDbt) -> Iterator[str]:
     for group in groups:
         if isinstance(group.task, DBTTask):
             yield from group.task.model_ids
+        if isinstance(group.task, FalHookTask):
+            # Is a before/after script
+            yield group.task.build_fal_script(fal_dbt).id
```

Comment:
NIT: add a sanity check (just in case)
```suggestion
            # Is a before/after script
            assert not group.task.is_hook
```